src/components/custom/dashboard/client/pages/diet-tracker/DayNavigation.jsx

```
@@ -0,0 +1,339 @@
+            const dayNumber = index + 1;
+            const status = getDayStatus(dayNumber);
+            const completion = getDayCompletion(dayNumber);
+            const isSelected = selectedDay === dayNumber;
+            const isCurrent = dayNumber === currentDay;
+
+            return (
+              <div
+                key={dayNumber}
+                className={getDayCardStyle(dayNumber)}
+                onClick={() => onSelectDay(dayNumber)}
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)